src/p2p/net_node.inl

```diff
@@ -1425,6 +1434,17 @@ namespace nodetool
     zone.m_notifier.on_handshake_complete(con->m_connection_id, con->m_is_income);
     zone.m_notifier.new_out_connection();
 
+    if (na.get_zone() == epee::net_utils::zone::tor) {
```

Comment:
Trivial, but bracing doesn't follow the style in this file or this patch.

Reply:
Will change.